scripts/install-dependencies.sh

```diff
@@ -71,7 +71,7 @@ if [ -z "${ANDROID_HOME}" ]; then
 fi
 
 # Install support-v4
-install_jar "com.android.support" "support-v4" "19.0.1" "${ANDROID_REPOSITORY}/com/android/support/support-v4/19.0.1/support-v4-19.0.1.jar"
+install_jar "com.android.support" "support-v4" "22.2.0" "$ANDROID_HOME/extras/android/support/v4/android-support-v4.jar"
```

Comment:
I'm a little hesitant to install the jar from the un-versioned source. Meaning, the jar could get updated to 23.0.0 and we'd still be installing it as version 22.2.0 (which could impact projects that use robolectric). Did you ever figure out why there are classes missing from the aar that are present in the jar?